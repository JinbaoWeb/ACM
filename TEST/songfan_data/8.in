300 193 107
1 7140
2 6345
3 9994
4 5174
5 6181
6 1905
7 4196
8 2692
9 6654
10 7670
11 6232
12 7295
13 2724
14 6535
15 7353
16 5377
17 4561
18 6793
19 4631
20 6521
21 4452
22 9757
23 2771
24 7875
25 7562
26 2031
27 6964
28 9516
29 2477
30 1984
31 8601
32 8804
33 2282
34 8117
35 1536
36 9989
37 1494
38 4417
39 5175
40 5812
41 2179
42 1905
43 7169
44 2277
45 5648
46 2727
47 6907
48 7988
49 7263
50 6618
51 5967
52 7953
53 8700
54 812
55 3785
56 6667
57 1514
58 7435
59 1656
60 368
61 5311
62 348
63 1463
64 5411
65 3555
66 2245
67 541
68 1152
69 5573
70 3369
71 2017
72 1325
73 7183
74 9882
75 3570
76 9105
77 5547
78 2778
79 7319
80 2601
81 5318
82 6306
83 2722
84 2568
85 9988
86 5879
87 2890
88 5263
89 3188
90 2105
91 1257
92 4365
93 5
94 2014
95 5635
96 601
97 8153
98 4719
99 2760
100 3438
101 5049
102 2593
103 8652
104 2870
105 634
106 3511
107 5125
108 2442
109 5998
110 5222
111 1620
112 4921
113 1502
114 7417
115 2248
116 3735
117 1494
118 2550
119 6575
120 6855
121 2897
122 1600
123 3439
124 1148
125 8599
126 4590
127 2020
128 1549
129 2740
130 4365
131 9914
132 2648
133 715
134 7905
135 1358
136 1496
137 5882
138 306
139 256
140 4137
141 4736
142 3972
143 4124
144 827
145 205
146 6039
147 2329
148 9685
149 4393
150 6561
151 3680
152 7581
153 5960
154 9848
155 7377
156 5845
157 7291
158 2133
159 2180
160 7304
161 5594
162 5605
163 9692
164 2837
165 3058
166 6033
167 7072
168 2890
169 5753
170 7841
171 6124
172 901
173 9194
174 7693
175 5333
176 840
177 8061
178 9396
179 6092
180 8603
181 5717
182 3936
183 3701
184 2678
185 1009
186 841
187 985
188 143
189 4747
190 2377
191 4489
192 7089
193 2773
3 0
5 1
9 1
9 5
9 0
17 0
22 6
23 2
26 4
28 1
29 9
31 1
37 5
37 1
41 5
42 0
42 1
43 0
43 1
46 3
48 2
51 1
52 3
52 0
54 2
55 1
56 1
56 0
58 2
58 0
61 0
64 3
64 2
65 2
68 2
69 2
69 0
70 1
71 0
73 3
73 0
73 0
75 0
79 6
79 0
79 0
80 0
81 2
82 1
82 0
84 1
85 0
87 0
87 0
99 2
105 11
105 6
107 0
112 5
115 4
116 2
118 0
118 1
119 1
120 1
120 4
121 0
121 1
121 0
122 0
122 0
123 1
128 4
136 3
136 3
138 1
139 5
140 2
140 0
141 1
142 1
144 1
144 1
145 0
149 0
150 4
151 2
152 1
162 10
163 2
164 1
164 0
167 3
171 4
172 1
173 0
173 0
173 1
174 0
176 2
178 0
178 1
180 4
180 0
181 1
184 0
190 4
