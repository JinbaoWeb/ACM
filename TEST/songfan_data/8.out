
3
36
85 131 74 154 22


163 148 28 178 173 76
32 53
103 180 31 125
97
34 177 48 52 134 24 170 174 10
152
25
58 114 155 15 79
160 12 157 49 73

43

1
192 167 27
47 120
18
56 9 50

119 150
14
20

2 82


11 5 171
179 146
166 109
51 153
137 86

156



40 169 181

45 95 162 161 69 77



64 16
175

81



61 88
110 39 4 107 101 112 189 141 98 19 126
17 191 21 38 149 92

130 7 140 143 142
182 55 116 183
151 75

65
106
123
100 70 89 165
121




87
168 104 164 78
193 23 99
129 46 13
83
8 184 132 80 102
84 118

29
108
190
147


33 44 115 66
159 41
158
90 26 127 71 94 30 6 42 59 111
122 128
35

57 113 136
37 117 63 135
72


91

68 124

185
187 172 186 176

144

54 133 105 96
